components/hacbs/jvm-build-service/base/self-binding.yaml

```diff
@@ -17,3 +17,15 @@ spec:
   - group: ""
     resource: events
     state: Accepted
+  - group: ""
+    identityHash: f67c7a5ce5c350f909090296ea42450d824759720824dc110de58aabe6873b22
```

Comment:
what is this hash is it random one? If so then it can be replaced by some message

Reply:
no it was the one employed/generated when I was last testing

what I'm try to tell you is that when I put some message like `pipielineIdentityHash` it was not getting replaced by the current boostrap script, as that is only looking for *apiexport*.yaml and not self-binding.yaml

all that said, this gets back to our on going discussion point the perm claims only existing in the `hacbs` workspace vs. the `redhat-hacbs` workspace, and the error message you said in slack we can ignore etc.

when I can bootstrap  a freshly created cluster without that perm claim in `redhat-hacbs` like you did over a week ago, then I'll officially be on board with that notion.  Until then, I still have lingering doubt on whether those error messages can be ignored.

Reply:
OK I **FINALLY** can see the same thing as @Michkov wrt 

```
$ KUBECONFIG=/tmp/ckcp-admin.kubeconfig oc ws redhat-hacbs
Warning: claim for services exported but not specified on APIBinding jvm-build-service
Add this claim to the APIBinding's Spec.
Warning: claim for services specified on APIBinding jvm-build-service but not accepted or rejected.
Warning: claim for deployments.apps:6e5884b86c4f84d860b24bcdfad9420f14c96c0f57f3ed54542e0f91d5b6ee3d exported but not specified on APIBinding jvm-build-service
Add this claim to the APIBinding's Spec.
Warning: claim for deployments.apps:6e5884b86c4f84d860b24bcdfad9420f14c96c0f57f3ed54542e0f91d5b6ee3d specified on APIBinding jvm-build-service but not accepted or rejected.
Warning: claim for pipelineruns.tekton.dev:6e5884b86c4f84d860b24bcdfad9420f14c96c0f57f3ed54542e0f91d5b6ee3d exported but not specified on APIBinding jvm-build-service
Add this claim to the APIBinding's Spec.
Warning: claim for pipelineruns.tekton.dev:6e5884b86c4f84d860b24bcdfad9420f14c96c0f57f3ed54542e0f91d5b6ee3d specified on APIBinding jvm-build-service but not accepted or rejected.
Current workspace is "root:users:zu:yc:kcp-admin:redhat-hacbs" (type "root:universal").
```

not having any bearing on the controller establishing watches 

I'll continue testing today, and at some point, see if 4Gi for ckcp works or not.  If it does not, I'll prune this PR for just the mem limit bump.  Otherwise, I'll close.